config/environments/production.rb

```diff
@@ -43,7 +43,7 @@
   # config.assume_ssl = true
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
+  config.force_ssl = ENV.fetch("SSL_ENABLE", "true") == "true"
```

Comment:
I would suggest using either `SSL_ENABLED` or `ENABLE_SSL` and keeping it consistent for the upcoming env vars

Reply:
You want me to have the variable `ENABLE_SSL`  confused. Please let me know I can update

Reply:
`ENABLE_SSL` or `SSL_ENABLED` => OK

`SSL_ENABLE ` => NOT OK